tasks/build-monthly-updates-as-json.js

```diff
@@ -114,6 +114,7 @@ function generateJson() {
         const imgContent = data['pim_announcement_img'] ? fs.readFileSync(path.dirname(file.path) + '/' + data['pim_announcement_img'], { encoding: 'base64' }) : null;
 
         let content = JSON.stringify({
+            'id':  'update_' + path.basename(file.path, '.md').replace('_', '-') + '_' + startDate,
```

Comment:
I thought it would be `update-` instead of `update_` : https://github.com/akeneo/actionable-product-data/issues/172